src/strings/PowerFxResources.en-US.resx

```diff
@@ -5158,4 +5158,15 @@
     <value>Can't delegate {0}: contains a behavior function '{1}'.</value>
     <comment>Warning message.</comment>
   </data>
+  <data name="AboutPlainText" xml:space="preserve">
+    <value>The PlainText function removes HTML and XML tags.</value>
+    <comment>Description of 'PlainText' function.</comment>
+  </data>
+  <data name="PlainTextArg1" xml:space="preserve">
+    <value>PlainText</value>
+    <comment>Function_parameter - First parameter for the PlainText function</comment>
```

Comment:
>Function_parameter - First parameter for the PlainText function [](http://example.com/codeflow?start=13&length=63)

"function_parameter - First argument of the PlainText function - the text to have its HTML and XML tags removed."

Reply:
Update here